fixtures/packaging/babel-standalone/dev.html

```diff
@@ -5,10 +5,8 @@
     <script src="https://unpkg.com/babel-standalone@6/babel.js"></script>
     <div id="container"></div>
     <script type="text/babel">
-      ReactDOM.render(
-        <h1>Hello World!</h1>,
-        document.getElementById('container')
-      );
+      ReactDOM.createRoot( document.getElementById('container')).render (
```

Comment:
I think the prettier configuration in our IDE is actually not helping to format and to remove whitespaces where ever required.

Suggesting the below changes
```
ReactDOM.createRoot(document.getElementById('container')).render(<h1>Hello World!</h1>);
```